<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientSocket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 223, 223);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBoxConnexionServeur">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 223, 223);</string>
      </property>
      <property name="title">
       <string>Connexion au serveur</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>561</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="labelAdresseServeur">
                <property name="text">
                 <string>Adresse IP du serveur : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelNumeroPortServeur">
                <property name="text">
                 <string>Num√©ro de port du serveur :</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEditAdresseIP"/>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxPort">
                <property name="minimum">
                 <number>1024</number>
                </property>
                <property name="maximum">
                 <number>65535</number>
                </property>
                <property name="value">
                 <number>8888</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonQuitter">
              <property name="text">
               <string>Quitter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonConnexion">
              <property name="text">
               <string>Connexion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelEtatConnexion">
            <property name="text">
             <string>Etat de la connexion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditEtat"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxInfosClient">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 223, 223);</string>
      </property>
      <property name="title">
       <string>Informations Client</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>561</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButtonOrdinateur">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Nom de l'ordinateur distant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUtilisateur">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Nom de l'utilisateur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonArchitecture">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Architecture processeur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOs">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>OS de l'ordinateur</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEditNomOrdinateur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNomUtilisateur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditArchitectureProcesseur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOSOrdinateur"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
